pages/api/_dbConnect.ts

```diff
@@ -0,0 +1,30 @@
+import { MongoClient } from 'mongodb'
+// Connection URI
+const prefix = process.env.NODE_ENV === 'development' ? '' : '+srv'
+const root = process.env.MONGO_INITDB_ROOT_USERNAME
+const pw = process.env.MONGO_INITDB_ROOT_PASSWORD
+const host =
+  process.env.NODE_ENV === 'development'
+    ? 'localhost:8080'
+    : process.env.MONGO_HOST
+const uri = `mongodb${prefix}://${root}:${pw}@${host}/?authSource=admin`
+
+const client = new MongoClient(uri)
+
+export const dbConnect = async (): Promise<MongoClient | undefined> => {
```

Comment:
This method connects to the database client and returns said client, allowing us to perform operations using the client, such as accessing databases or adding data to the database. Essentially, the client is our access point to modify the database.